GameDefinition/Content/ModeKind.hs

```diff
@@ -248,10 +248,10 @@ safari = ModeKind  -- Easter egg available only via screensaver
   , mcaves  = cavesSafari
   , mendMsg = []
   , mrules  = T.unlines $
-      [ "* three levels"
-      , "* many friends that move concurrently, many foes"
-      , "* avoid losses"
-      , "* find exit and escape ASAP"
+      [ "* Three levels"
+      , "* Many allies capable of concurrent action vs. many enemies capable of concurrent action"
```

Comment:
How did I know? Even a broken clock is right twice a day 😂. I agree with switching allies with teammates since ally has another meaning in game. So with the above recommended changes this would look like
 > "Many teammates capable of concurrent action vs. Many enemies capable of concurrent action"

Reply:
This is correct but, again, isn't the second part too verbose? In the same case earlier you only mention "Many enemies", which is fine, I think (by default enemies are capable of concurrent action; only humans by default are capable of concurrent melee exclusively).